### PR TITLE
 fix CSW xml types

### PR DESCRIPTION
**Pull request by nf-s**
### Fix CSW xml types

See comment below

### Checklist

-   [x] I don't think tests are required
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/Models/CswCatalogGroup.ts [hunk 1]
```diff
@@ -325,7 +325,10 @@ class CswStratum extends LoadableStratum(CswCatalogGroupTraits) {
       records.push(...(json?.SearchResults?.Record ?? []));
 
       // Get next start position - or stop pageing
-      const nextRecord = json?.SearchResults?.nextRecord;
+      const nextRecord =
+        typeof json?.SearchResults?.nextRecord === "string"
+          ? parseInt(json?.SearchResults?.nextRecord ?? "0")
+          : json?.SearchResults?.nextRecord;
```
Comment (nf-s) [inline lib/Models/CswCatalogGroup.ts]: When paging for CSW records, we use the `nextRecord` ID in the response to set `startPosition`. 

This was a `string`, but should have been a `number` - so we were getting this error:

```
startPosition must be an integer greater than zero.
```

**Comment by KeyboardSounds** [comment]
Github won't let me add comments because it's outside the part of the code you changed, but I can see places in the code that assume `numberOfRecordsMatched` and `numberOfRecordsReturned` are numbers but they could be strings. Is that okay?

**Comment by nf-s** [comment]
> Github won't let me add comments because it's outside the part of the code you changed, but I can see places in the code that assume `numberOfRecordsMatched` and `numberOfRecordsReturned` are numbers but they could be strings. Is that okay?

Fixed!

**Review by KeyboardSounds — APPROVED** [review]
Once you make sure this PR isn't the thing breaking CI, merge away.

**Pull request merged**